src/app/api/event/route.js

```diff
@@ -1,6 +1,80 @@
 import { prisma } from '@/lib/db';
 
-export async function GET() {
-  const events = await prisma.event.findMany({ include: { tags: true } });
-  return Response.json(events);
+export async function GET(req) {
+  const today = new Date();
+  const currentYear = today.getFullYear();
+
+  try {
+    const url = new URL(req.url);
+    const take = url.searchParams.get('take');
+    const lastCursor = url.searchParams.get('lastCursor');
+    const queryMonth = url.searchParams.get('month');
+
+    const filteredQueryMonth = parseInt(queryMonth < 10 ? `0${queryMonth}` : queryMonth, 10);
```

Comment:
Same for default radix, we can safely remove it

Reply:
> @Yivgha is there any sense for this ternaries operator? You parsing string immediately after padding it with zero

yeah, because from query we get number without 0 and to pass month to new Date we should have it. if month is 10+ we don't need it.

Reply:
@Yivgha sounds reasonable, but this code you are doing opposite thing.

Imagine that you have queryMonth = 4.
1. parseInt(4 < 10 ? `0${4}` : 4, 10);
2. 4 is smaller than 10, so we have parseInt( `04`, 10);
3. When parseInt parses string '04' we will get number 4.

So in this line we are not doing anything useful. It's the same if we just wrote `parseInt(queryMonth, 10)`.

p.s. btw, you are passing after that number without 0 and everything works

Reply:
changed